### ISSUE TITLE
header corrupted

### ISSUE THREAD

**Issue by wgzhao** [trinitronx/lzop-file#1]
I  run the  example code ,write a leo-format file ,I got a error when i decompress it using lzop .the detail is the following:

```
lzop -t my_test_file.lzo
lzop: my_test_file.lzo: header corrupted (checksum error)
```

the following is my code:

``` ruby
require 'lzoruby'
require 'lzop-file'

uncompressed_file_data = "Hello World\n" * 100
my_test_file = LZOP::File.new( '/tmp/my_test_file.lzo' )
my_test_file.write( uncompressed_file_data )
```

**Comment by lorgio** [comment]
+1

**Comment by trinitronx** [comment]
Yes, this is listed in the README under known issues.  I started this project as a hackathon and didn't have time to finish figuring out the file header and checksum parts.  So it's not really complete and is just a beta quality project for now.  I haven't found the need to use this or work on it again since the hackathon, so that's where it stands at this point.  c'est la vie
